src/content/en/fundamentals/media/fast-playback-with-video-preload.md

```diff
@@ -0,0 +1,492 @@
+
+# Fast Playback with Video Preload {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque suscipit
+nibh nec feugiat ornare. Nam tristique, magna non pharetra mollis, diam elit
+faucibus nibh, ut condimentum diam metus vel dolor. Integer facilisis magna at
+ex bibendum luctus. In eu tincidunt ipsum. Duis scelerisque, tortor quis
+condimentum finibus, est neque mollis leo, at imperdiet justo elit dapibus
+nisl. Integer gravida dapibus turpis, vel scelerisque dolor scelerisque eget.
+Sed dolor lectus, tempor vulputate bibendum eget, accumsan lobortis ex.
+
+<video controls controlsList="nodownload" muted playsinline style="width: 100%">
+  <source src="https://storage.googleapis.com/webfundamentals-assets/videos/video-preload-hero.webm#t=0.7"
+          type="video/webm">
+  <source src="https://storage.googleapis.com/webfundamentals-assets/videos/video-preload-hero.mp4#t=1"
+          type="video/mp4">
+</video>
+
+Unless specified otherwise, this article also applies to the audio element.
+
+## Video preload attribute
+
+If the video source is an unique file hosted on a web server, you may want to
+use the video `preload` attribute to provide a hint to the browser as to [how
+much information or content to preload]. This means [Media Source Extensions
+(MSE)] is not compatible with `preload`.
+
+Resource fetching will start only when initial HTML document has been
+completely loaded and parsed (eg. `DOMContentLoaded` event) while the very
+different `Load` event will be fired when resource has actually been fetched.
+
+<figure class="TODO">
+  <img src="/web/fundamentals/media/images/video-preload/video-preload.svg">
+</figure>
+
+Setting `preload` value to `metadata` indicates that the user is not expected
+to need the video, but that fetching its metadata (dimensions, first frame,
+track list, duration, and so on) is desirable.
+
+```
+<video id="video" preload="metadata" src="file.mp4" controls></video>
+
+<script>
+  video.addEventListener('progress', function() {
+    if (video.buffered.length === 0) return;
+
+    var bufferedSeconds = video.buffered.end(0) - video.buffered.start(0);
+    console.log(bufferedSeconds + ' seconds of video are ready to play!');
+  });
+</script>
+```
+
+With `preload` value set to `auto`, we're saying that downloading the entire
+video right away is considered desirable.
+
+```
+<video id="video" preload="auto" src="file.mp4" controls></video>
+
+<script>
+  video.addEventListener('progress', function() {
+    if (video.buffered.length === 0) return;
+
+    var bufferedSeconds = video.buffered.end(0) - video.buffered.start(0);
+    console.log(bufferedSeconds + ' seconds of video are ready to play!');
+  });
+</script>
+```
+
+There are some caveats though. As this is just a hint, browser may completely
+ignore the `preload` attribute. At the time of writing, here are some rules
+applied in Chrome:
```

Comment:
The `/web/media` section is a living document and *will* be updated hence why I think it's good to give details when we can. I'll address your feedback below.